index.html

```diff
@@ -1,286 +1,430 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>  
-  <link rel="stylesheet" href="./css/style.css">
-  <link rel="stylesheet" href="./css/style-desktop.css">
-  <title>Portfolio</title>
-</head>
-<body>
-  <header class="header--section">
-    <a href="#" class="logo">Jos</a>
-    <nav class="nav--container">
-      <ul class="nav-menu d-flex">
-        <li class="nav-link"><a href="#works">Portoflio</a></li>
-        <li class="nav-link"><a href="#about">About</a></li>
-        <li class="nav-link"><a href="#contact">Contact</a></li>
-      </ul>
-    </nav>
-    <span  class= "nav-humburger" id="humburger"><i class="fa humburger fa-bars"></i></span>
-  </header>
-  <main class="main--container">
-    <section class="main--section">
-      <div class="section--title">
-        <h1>Hey there. I'm Jos Kalenda <br>
-          <span>I'm a software developer</span>
-        </h1>
-      </div>
-      <div class="paragraphe">
-        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
-      <div class="section--icons">
-        <ul class="social--icons d-flex "> 
-          <li>
-            <a href="https://github.com/joskalenda">
-              <img src="./assets/icons/github.png" alt="github account">
-            </a>
-          </li>
-
-          <li>
-            <a href="https://www.linkedin.com/in/jos-kalenda-908069205">
-              <img src="./assets/icons/linkedin.png" alt="linkedin account">
-            </a>
-          </li>
-          <li>
-            <a href="**">
-              <img src="./assets/icons/peace.png" alt="peace">
-            </a>
-          </li>
-
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>  
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Jos - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <a href="#" class="logo">Jos</a>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
-            <a href="https://www.twitter.com/in/joskalenda">
-              <img src="./assets/icons/twitter.png" alt="twitter account">
-            </a>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
           </li>
```

Comment:
As requested by the previous reviewer, it is vital to at least make a copy of you website look like the Figma design to improve the UI, consider adding the bottom borders on each nav links